backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/UsageNodeDescriptionProvider.java

```diff
@@ -73,16 +73,13 @@ public void link(DiagramDescription diagramDescription, IViewDiagramElementFinde
 
         var allTargetNodeDescriptions = new ArrayList<NodeDescription>();
 
-        GeneralViewDiagramDescriptionProvider.DEFINITIONS.forEach(definition -> {
-            var optNodeDescription = cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(definition));
-            allTargetNodeDescriptions.add(optNodeDescription.get());
+        GeneralViewDiagramDescriptionProvider.TOOL_SECTIONS.forEach((sectionName, elements) -> {
```

Comment:
Same remark than for the DefinitionNodeDescriptionProvider.

Reply:
Reverted